SnowplowTracker/SnowplowTracker/Emitters/AsyncEmitter.cs

```diff
@@ -182,8 +190,14 @@ public class AsyncEmitter : AbstractEmitter {
 							failure += result.rowIds.Count;
 						}
 					}
-					
-					eventStore.DeleteEvents(eventsToDelete);
+					if (emitLock != null)
+		    			{
```

Comment:
small nitpick, but I'd prefer if we keep the code style consistent with the braces here

Reply:
not nitpick at all.

c# style guide says braces below, much to the chagrin of the java community :)  I can fix this up for you

Reply:
Fixed.

Reply:
btw is there a good option for linting and formatting the entire codebase? I don't do too much C# development so not too familiar with the tooling.